### ISSUE TITLE
Can't get Turpial version from CLI when another instance is running

### ISSUE THREAD

**Issue by Ryuno-Ki** [satanas/Turpial#366]
I've started a Turpial instance (v3.5.9-dev) and tried to get its version via `turpial --version` which yield this:

<pre><code>$ turpial --version
** Message: pygobject_register_sinkfunc is deprecated (GstObject)
DEBUG::Using gst as sound system
/usr/lib/python2.7/site-packages/gtk-2.0/gtk/__init__.py:127: RuntimeWarning: PyOS_InputHook is not available for interactive use of PyGTK
  set_interactive(1)
ERROR:Sys:Another instance is already running</code></pre>

**Comment by satanas** [comment]
Yes, because Turpial avoids to execute multiple instances of the app. You can check the version on the 'About Turpial' menu. I'll check if I can avoid the singleton validation for this option

**Comment by Ryuno-Ki** [comment]
While you're at it, I'd expect `--help` to work aside a singleton, too.

**Comment by satanas** [comment]
Roger that
